Website update after version stream release is broken

Specifically the `gen-crd-api-reference-docs` with this:

```
/workspace/go/bin/gen-crd-api-reference-docs -config "docs/refdocs/config.json" \
-template-dir docs/refdocs/templates \
    -api-dir "./pkg/apis/jenkins.io" \
    -out-file docs/apidocs.md
I0616 15:22:29.081513     313 main.go:126] parsing go packages in directory ./pkg/apis/jenkins.io
W0616 15:22:35.962219     313 parse.go:239] Ignoring child directory github.com/jenkins-x/**/pkg/apis/jenkins.io/v1: unable to import "github.com/jenkins-x/**/pkg/apis/jenkins.io/v1": go/build: importGo github.com/jenkins-x/**/pkg/apis/jenkins.io/v1: exit status 1
go: finding github.com/jenkins-x/** v1.3.1119
go: downloading github.com/jenkins-x/** v1.3.1119
go: extracting github.com/jenkins-x/** v1.3.1119
go: github.com/jenkins-x/**@v1.3.1119 requires
	github.com/antham/chyle@v1.4.0 requires
	github.com/spf13/cobra@v0.0.0-00000000000000-c46add8a6528: reading http://jenkins-x-athens-proxy/github.com/spf13/cobra/@v/v0.0.0-00000000000000-c46add8a6528.mod: 404 Not Found

W0616 15:22:40.253095     313 parse.go:239] Ignoring child directory github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data: unable to import "github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data": go/build: importGo github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data: exit status 1
can't load package: package github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data: module github.com/jenkins-x/**@latest found (v1.3.1119), but does not contain package github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data

W0616 15:22:44.263172     313 parse.go:239] Ignoring child directory github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data/environment: unable to import "github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data/environment": go/build: importGo github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data/environment: exit status 1
can't load package: package github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data/environment: module github.com/jenkins-x/**@latest found (v1.3.1119), but does not contain package github.com/jenkins-x/**/pkg/apis/jenkins.io/v1/test_data/environment

F0616 15:22:44.263336     313 main.go:132] no API packages found in ./pkg/apis/jenkins.io
make: *** [generate-api-refdocs] Error 255
```

This is called from https://github.com/jenkins-x/jenkins-x-versions/blob/master/jx/scripts/update-jx-website.sh#L60 in [the release pipeline for the version stream](https://github.com/jenkins-x/jenkins-x-versions/blob/master/jenkins-x.yml#L46-L48), and because it's failing at this point, it's never actually pushing the updated CLI docs (which do generate correctly before this point) to the doc site.

--- a/Makefile.codegen
+++ b/Makefile.codegen
@@ -66,7 +66,7 @@ generate-refdocs: generate-api-refdocs generate-config-refdocs
 generate-api-refdocs: install-refdocs
 	${GOPATH}/bin/gen-crd-api-reference-docs -config "docs/refdocs/config.json" \
 	-template-dir docs/refdocs/templates \
-    -api-dir "./pkg/apis/jenkins.io" \
+    -api-dir "./pkg/apis/jenkins.io/v1" \
     -out-file docs/apidocs.md
 
 generate-config-refdocs:

--- a/hack/generate.sh
+++ b/hack/generate.sh
@@ -11,7 +11,9 @@ echo "Running validation scripts..."
 scripts=(
     "make verify-generation-complete"
     "make generate-docs"
+    "make generate-refdocs"
 )
+
 fail=0
 for s in "${scripts[@]}"; do
     echo "RUN ${s}"

